### ISSUE TITLE
Add custom error page

### ISSUE THREAD

**Issue by pragmaticivan** [wedeploy/wedeploy.com#503]
Right now it only shows a blank page with little texts.

https://cl.ly/09072n2k0F1m

**Comment by jonnilundy** [comment]
@albertmourato could you look into this and discover how we can set up error pages in Magnet? @vitorfernandes can supply the error pages when its ready.

**Comment by vitorfernandes** [comment]
where exactly is this? I missing context.

**Comment by pragmaticivan** [comment]
If someone access /xxxx that's the page.

**Comment by vitorfernandes** [comment]
interesting, shouldn't be 404?

**Comment by pragmaticivan** [comment]
yes, the task is to add a custom error page like the one you designed, this one would be 404.

**Comment by vitorfernandes** [comment]
sounds good then. 👍
